cmd/rm-main.go

```diff
@@ -182,11 +180,7 @@ func (r rmMessage) String() string {
 		msg = console.Colorize("Remove", fmt.Sprintf("Creating delete marker `%s`", r.Key))
 	}
 	if r.VersionID != "" {
-		if !r.ModTime.IsZero() {
-			msg += fmt.Sprintf(" (versionId=%s, modTime=%s)", r.VersionID, r.ModTime)
```

Comment:
why are we not interested in printing modTime anymore?

Reply:
> why are we not interested in printing modTime anymore?

Because it is buggy. It doesn't show the correct information sometimes.
```
                              select {
                                case contentCh <- content:
                                case result := <-resultCh:
                                        // using a mix of result & content variables
```

The old code was mixing the use of the element that we sent to deletion and the result deletion element (content & result above. However this is wrong, result doesn't always correspond to content here.

Basicaly deletion is asynchorounous in mc perspective.